lib/tasks/content_change_email.rake

```diff
@@ -1,8 +1,18 @@
 namespace :report do
-  desc "Produce a report on sent, pending and failed emails for a given content change"
-  task :content_change_email_status_count, [:id] => :environment do |_t, args|
-    content_change = ContentChange.find(args[:id])
-    Reports::ContentChangeEmailStatusCount.call(content_change)
+  desc "Produce a report on sent, pending and failed emails for given content change id or ids"
+  task content_change_email_status_count: :environment do |_t, args|
+    ids = args.extras
+    if ids.present?
+      ids.each do |id|
+        content_change = ContentChange.find(id)
+        Reports::ContentChangeEmailStatusCount.call(content_change)
+      end
+    else
+      puts "At least one ContentChange id must be given."
```

Comment:
Perhaps it's better to put this in the `desc` above so anyone looking at the documentation for this task would get this information without having to run it.

Reply:
I'll write a longer description but I want it here too so if someone happens to not pass an id, at least has a helpful message.